metl/models.py

```diff
@@ -4,6 +4,8 @@
 import enum
 from os.path import isfile
 from typing import List, Tuple, Optional
+from biopandas.pdb import PandasPdb
+import os
```

Comment:
Are these imports used?

Reply:
No, I was originally testing validation in the forward pass to see how bad it was, and this was a left over from that. I'll remove it, good catch